Symfony/CS/Fixer/PSR0/Psr0Fixer.php

```diff
@@ -161,6 +160,12 @@ public function supports(\SplFileInfo $file)
             return false;
         }
 
+        $tokens = token_get_all(sprintf('<?php %s', $filenameParts[0]));
```

Comment:
:+1: 
should these be added to the list of predefined constants or do you want those on PHP7 only http://php.net/manual/en/reserved.other-reserved-words.php ?
btw. the same checks added to the class should be done for (parts of)  the namespace.